### ISSUE TITLE
No more support Foxy version?

### ISSUE THREAD

**Issue by james-yoo** [Greenroom-Robotics/rclUE#9]
@russkel 

According to the change in the README file, does the Whiskey branch no longer support the foxy version? Or will it be possible to use with rcutils patch?

**Comment by russkel** [comment]
Due to the patches I made to use the Unreal Engine memory allocator, it would need the following PR, which hasn't been merged into foxy yet: https://github.com/ros2/rosidl/pull/604. Galactic is the same (https://github.com/ros2/rosidl/pull/603)

This would then require a local compile of all the interfaces you use. I believe it would continue to work as expected.

I have been plagued by memory issues since I forked rclUE, and I think I have finally fixed the problems.
